decidim-proposals/lib/decidim/proposals/proposal_serializer.rb

```diff
@@ -93,6 +95,16 @@ def original_proposal_url
 
         Decidim::ResourceLocatorPresenter.new(proposal.amendable).url
       end
+
+      def plain_text_body
+        proposal.body.transform_values do |v|
+          if v.is_a? Hash
+            v
+          else
+            decidim_sanitize(v, strip_tags: true)
```

Comment:
The problem with using `strip_tags` is that we are losing the paragraph information and line breaks.

For example, if you enable rich text editor and create a proposal like this (this is how it is saved):

```html
<p>First paragraph.</p><p>And we have <strong>some HTML content</strong> here.</p><ul><li>Here</li><li>Is</li><li>A</li><li>List</li><li class=\"ql-indent-1\">With</li><li class=\"ql-indent-2\">Some</li><li class=\"ql-indent-3\">Sub items</li></ul><p>Does it <strong><em>export </em></strong><em>correctly</em>?</p><ol><li>Also we may</li><li>have</li><li>ordered</li></ol><ul><li class=\"ql-indent-1\">with</li><li class=\"ql-indent-2\">some</li><li class=\"ql-indent-3\">sub-items</li></ul><ol><li>lists</li></ol>
```

This will export as:

```
This is a testing proposal for the autocomplete.And we have some HTML content here.
Here
Is
A
List
With
Some
Sub items
Does it export correctly?
Also we may
have
ordered

with
some
sub-items
lists
```

Note particularly how the first two paragraphs are sticked to each other.

Fortunate thing is that we already have [`premailer` as a Decidim dependency](https://github.com/decidim/decidim/blob/75c1d19e2a298e101ad382582309b6982191268d/Gemfile.lock#L583), so we could use this module:
https://github.com/alexdunae/premailer/blob/be30faeeec0a01444fcac9f2c03d558e1855082b/lib/premailer/html_to_plain_text.rb#L5

With that, this would convert to:

```
This is a testing proposal for the autocomplete.                                                                     
                                                                                                                     
And we have some HTML content here.                                                                                  
                                                                                                                     
* Here                                                                                                               
* Is                                                                                                                 
* A                                                                                                                  
* List                                           
* With                                           
* Some                                           
* Sub items                                      
Does it export correctly?                        
                                                 
* Also we may
* have
* ordered
* with
* some
* sub-items
* lists
```

It's still not perfect due to the way how QuillJS handles lists but I'd say still better.

Reply:
This was already fixed at a61c0f6425435c98d2d0716e9b5e32d9d93cf9e8.